js/script.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+$(function() {
+
+    function checkUser(e, elements, users) {
+        setTimeout(function () {
+            let cursorPosition = elements.textArea.prop("selectionStart");
+            if(cursorPosition){
+                let value = elements.textArea.val();
+                let startPos = getStartPosition(value, cursorPosition-1);
+                if(startPos){
+                    let text = value.substring(startPos, cursorPosition);
+                      return  find(text, users, elements);
+                }
+                return elements.showUsers.empty();
+                }
+        }, 0);
+
+    }
+
+    function getStartPosition(str, pos) {
+        if (!str.substring(pos, pos + 1).match(/[A-Za-z]/)) {
+            return str.substring(pos, pos + 1) === '@' ? pos + 1: null;
+        } else if (pos === 0) {
+            return 0
+        } else {
+            return getStartPosition(str, pos - 1);
+        }
+    }
+
+    function find(user, users, elements) {
+        elements.showUsers.empty();
```

Comment:
@lianatumikyan please made the local variable for elements.showUsers (you use this more than one time, so will be correct if you will make it as local).